resources/api/mint-api.yaml

```diff
@@ -333,7 +333,7 @@ definitions:
         items:
           type: string
           example: example-org-stups-mint
-          pattern: "^[a-z0-9][a-z0-9-.]{1,61}[a-z0-9]$"
+          pattern: "^(gs://)?[a-z0-9][a-z0-9-.]{1,61}[a-z0-9]$"
```

Comment:
this regex does not work according to regex101. What about this one? https://regex101.com/r/hY9lN8/2